salt/utils/event.py

```diff
@@ -586,6 +591,9 @@ def __del__(self):
             self.destroy()
         except Exception as ex:
             log.debug(ex)
+            # as this method can be called when logging is already teared down
+            # just try to stdout it
+            traceback.print_exc(file=sys.stderr)
```

Comment:
If we can't trust that logging is there, why do we think we can trust traceback? If the concern is that it logs too much we can just change this to a "pass" instead of a log